config/custom-environment-variables.js

```diff
@@ -19,9 +19,15 @@ module.exports = {
     __format: 'boolean'
   },
 
+  githubApi: {
+    baseUrl: 'GITHUB_API_BASE_URL',
+    org: 'GITHUB_ORGANISATION'
+  },
+
   githubOauth: {
     clientId: 'GITHUB_CLIENT_ID',
-    clientSecret: 'GITHUB_CLIENT_SECRET'
+    clientSecret: 'GITHUB_CLIENT_SECRET',
+    accessToken: '<accessToken>'
```

Comment:
WHat is the access token? And how do we get it? Aren't we using client Id and client secret for making requests?

Reply:
Please refer my reply to your earlier review
Posted on discord as well

Reply:
Done. Authorizing now using clientId and clientSecret. Please check